src/renderer/renderScene.ts

```diff
@@ -481,13 +483,24 @@ export const renderScene = (
     context.stroke();
 
     const username = sceneState.remotePointerUsernames[clientId];
-    const usernameAndIdleState = `${username ? `${username} ` : ""}${
-      userState === UserIdleState.AWAY
-        ? "⚫️"
-        : userState === UserIdleState.IDLE
-        ? "💤"
-        : "🟢"
-    }`;
+    let usernameAndIdleState;
+    if (hasEmojiSupport) {
+      usernameAndIdleState = `${username ? `${username} ` : ""}${
+        userState === UserIdleState.AWAY
+          ? "⚫️"
+          : userState === UserIdleState.IDLE
+          ? "💤"
+          : "🟢"
+      }`;
+    } else {
+      usernameAndIdleState = `${username ? `${username}` : ""}${
+        userState === UserIdleState.AWAY
+          ? ` (${UserIdleState.AWAY})`
+          : userState === UserIdleState.IDLE
+          ? ` (${UserIdleState.IDLE})`
+          : ""
+      }`;
+    }
```

Comment:
```ts
    let usernameAndIdleState = `${username ? `${username} ` : ""}${
      userState === UserIdleState.AWAY
        ? "⚫️"
        : userState === UserIdleState.IDLE
        ? "💤"
        : "🟢"
    }`;
    if (!hasEmojiSupport) {
      usernameAndIdleState = `${username ? `${username}` : ""}${
        userState === UserIdleState.AWAY
          ? ` (${UserIdleState.AWAY})`
          : userState === UserIdleState.IDLE
          ? ` (${UserIdleState.IDLE})`
          : ""
      }`;
    }
```

I'm more of this kind of guy :) But maybe not popular style :) (Less lines, no else.. most common case first)

/cc @dwelle

Reply:
No need to change it.. just putting it out there :)